code/game/mecha/equipment/tools/work_tools.dm

```diff
@@ -330,8 +330,6 @@
 	name = "cable layer"
 	desc = "Equipment for engineering exosuits. Lays cable along the exosuit's path."
 	icon_state = "mecha_wire"
-	var/datum/event/event
```

Comment:
this one is especially concerning

Reply:
Oh god, I remember this one yeah. A mecha tool trying to make an event... forgot to remove it back when I fixed that shit 😅

Reply:
But why though.